python/erdos/__init__.py

```diff
@@ -18,6 +18,8 @@
 import erdos.utils
 
 _num_py_operators = 0
+node_addrs = dict()
```

Comment:
Rename to `node_to_address`.

Could you add a comment that this maps each Node ID to an addresses and a port offset?